grpc/proto/space.proto

```diff
@@ -584,6 +584,7 @@ message BackupKeysByPassphraseResponse {}
 message RecoverKeysByPassphraseRequest {
   string uuid = 1;
   string passphrase = 2;
+  KeyBackupType type = 3;
```

Comment:
You should also update the `enum KeyBackupType` definition of the proto file to contain the new fields

Reply:
Good catch, fixed in [8f5fafc](https://github.com/FleekHQ/space-daemon/pull/260/commits/8f5fafc86fe33d2152906df537be2b92ef54bf80)